### PR TITLE
Rails toast redesign

### PR DESCRIPTION
**Pull request by gowsik-ragunath**
## Notion card

https://www.notion.so/saeloun/Toast-message-in-React-and-rails-should-be-same-b70c5924a7454544bafbda6e1c564a08

## Summary

Redesigned the rails toast notification to match react toast design.

## Preview

https://www.loom.com/share/0fd6fcaed9624ceebc97e98e4e05a15f

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [ ] I have manually tested all workflows
- [ ] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

File: app/javascript/stylesheets/application.scss [hunk 1]
```diff
@@ -491,3 +491,71 @@ body {
     letter-spacing: 0px;
   }
 }
+
+#toast-container {
```
Comment (akhilgkrishnan) [inline app/javascript/stylesheets/application.scss:495]: can you move this to toast.scss

Comment (gowsik-ragunath) [reply app/javascript/stylesheets/application.scss]: Moved to toast.scss.

**Comment by supriya3105** [comment]
@gowsik-ragunath CI tests are failing on this PR. Please fix it.

**Comment by gowsik-ragunath** [comment]
@supriya3105 CI tests are passing now.

**Review by apoorv1316 — APPROVED** [review]

**Pull request merged**